src/renderer/App.css

```diff
@@ -44,11 +44,12 @@
   }
 
   .banner {
-    width: 90%;
+    width: 80%;
+    margin-top: 60px;
   }
 
   .flexBox {
     margin-top: 60px;
     margin-bottom: -75px;
   }
-}
+}
```

Comment:
I think we do have `.editorconfig` to ensure there is a final newline char...
Maybe your editor/IDE need to be setup to support it
https://github.com/FreeTubeApp/FreeTube/blob/development/.editorconfig#L9
https://editorconfig.org/#download

Reply:
Updated.